pkg/kapp/app/recorded_app.go

```diff
@@ -80,9 +83,31 @@ func (a *RecordedApp) UpdateUsedGVs(gvs []schema.GroupVersion) error {
 func (a *RecordedApp) CreateOrUpdate(labels map[string]string) error {
 	defer a.logger.DebugFunc("CreateOrUpdate").Finish()
 
+	if os.Getenv("USE_EXISTING_CONFIGMAP_NAME") != "True" {
```

Comment:
Perhaps the variable should be renamed as I would expect `USE_EXISTING_CONFIGMAP_NAME=True` to still work if I am `kapp deploying` a new app with the suffix.

If we make the behaviour of `USE_EXISTING_CONFIGMAP_NAME` that it will only work given the old name, maybe something like `USE_OLD_CONFIGMAP_NAME`

Reply:
I went ahead and added the case where it's True and also renamed it.